cmd/trade.go

```diff
@@ -210,8 +215,8 @@ func init() {
 
 		strategyFillHandlers, e := strat.GetFillHandlers()
 		if e != nil {
-			log.Println()
-			log.Printf("problem encountered while instantiating the fill tracker: %s\n", e)
+			l.Info("")
+			l.Infof("problem encountered while instantiating the fill tracker: %s\n", e)
```

Comment:
this should follow the same pattern as the logging of the monitoring server to keep things consistent. either change both to a new pattern or none.